Source/Build Specs/MeasurementLink Generator.vipb

```diff
@@ -30,11 +30,11 @@
     </Custom_Action_VIs>
     <VI_Package_Configuration_File>MeasurementLink Generator.vipc</VI_Package_Configuration_File>
     <Description>
-      <One_Line_Description_Summary/>
-      <Description>This is the MeasurementLink Plug-in Generator package.</Description>
+      <One_Line_Description_Summary>Contains a tool which generates a plugin template</One_Line_Description_Summary>
+      <Description>Contains a tool which generates a measurementlink plugin template.  Accessable from the LabVIEW tools menu.</Description>
```

Comment:
Did I mention I really dislike the MeasurementLink branded VI icon that is shown for the package. 😃

Reply:
Updated summary.  Do you have an alternate suggestion for the icon @jasonmreding ?

Reply:
Not really, at least not anything I have a ready made icon I could give you. For the client side libraries, this is rough iconography for the public APIs.

![image](https://github.com/ni/measurementlink-labview/assets/103200169/172b55be-c0f4-489a-ab7e-96aedff81ed7)

We could reuse one of those banners, but I'm not sure what icon to put in the main part unless you like just leaving it a field of white space.

Otherwise, I would almost rather use a generic NI icon/logo like you get with the task bar icon with the UI editor. However, I don't know how easy that would be to convert into a 32x32 icon. Just browsing through the [NI packages online](https://www.vipm.io/publisher/ni/), it doesn't look like we've put a whole lot of effort into this in the past. Maybe just using the MLink banner as shown above with some basic icon/text would be sufficient? Here's a shot at some alternatives using text and basic glyphs that already exist in the default icon library used by LV. If you like any of these options better, feel free to steal them.

![image](https://github.com/ni/measurementlink-labview/assets/103200169/362cc2f0-f690-4453-a461-f9e42d36c2f5)

Reply:
@jasonmreding I'm inclined to leave it as-is.  None of the other icons seem unique enough to gather the purpose at a glance.  Technically that is the official icon from our marketing folks.